### PR TITLE
before review

### PR DESCRIPTION
**Pull request by channamakover**

### CODE REVIEW

File: 1-Vanilla/index.html [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+<html lang="en">
+<head>
+    <link rel="stylesheet" href="styles.css">
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <header class="header">
```
Comment (tombenezra) [inline 1-Vanilla/index.html]: unused? remove if so

File: 1-Vanilla/index.html [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    </header>
+    <div class="container">
+        
+        <div class="ul-container">
```
Comment (tombenezra) [inline 1-Vanilla/index.html]: better not to name class with their corresponding html element

File: 1-Vanilla/index.html [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+            <h2>to-do list</h2>
+            <div id="add-button-container" class="add-task-div">
+                <input type="text" id="task-title-input" class="to-do-task-title"/>
+                <!-- <button class="button-style" id="add-button">add</button> -->
```
Comment (tombenezra) [inline 1-Vanilla/index.html]: remove comments

File: 1-Vanilla/styles.css [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+}
+.ul-container{
+    display:flex;
+    justify-content:center;
+    flex-direction:column;
+    text-align:center;
+    border:2px solid;
+    border-radius: 40px;
+    width: 50%;
+    height:100vm;
```
Comment (tombenezra) [inline 1-Vanilla/styles.css]:
```suggestion
    height:100vh;
```

File: 1-Vanilla/styles.css [hunk 5]
```diff
@@ -0,0 +1,66 @@
+body{
```
Comment (tombenezra) [inline 1-Vanilla/styles.css]: run formatter on this file

**Review by tombenezra — CHANGES_REQUESTED** [review]

File: 1-Vanilla/index.html [hunk 6]
```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <link rel="stylesheet" href="styles.css" />
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
```
Comment (ofirdagan) [inline 1-Vanilla/index.html]: what is this line. what does each param/value means

File: 1-Vanilla/index.html [hunk 7]
```diff
@@ -0,0 +1,23 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
```
Comment (ofirdagan) [inline 1-Vanilla/index.html]: what is this line. what does each param/value means

File: 1-Vanilla/index.html [hunk 8]
```diff
@@ -0,0 +1,23 @@
...
+    <title>Document</title>
```
Comment (ofirdagan) [inline 1-Vanilla/index.html]: u might want to give it a better title ;)

File: 1-Vanilla/toDoListApp.js [hunk 9]
```diff
@@ -0,0 +1,112 @@
...
+
+  function editElementButtonHandler(id) {
+    let task = document.getElementById(id);
+    let newInput = document.getElementById("task-title-input");
+    let taskTitle = task.querySelector("#task-title");
+    taskTitle.textContent = newInput.value;
+  }
+  function createButtonElement(className, eventType, callBack = () => {}, params = "") {
+    let buttonElement = createHtmlElement("button", className);
+    buttonElement.addEventListener(eventType, () => {
```
Comment (ofirdagan) [inline 1-Vanilla/toDoListApp.js]: I liked the fact that you took the logic of building a button element and put it together in a single function.
for each event there's the `event` param that your'e currently "throwing away".  make sure to integrate with it

File: 1-Vanilla/index.html [hunk 10]
```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
```
Comment (ofirdagan) [inline 1-Vanilla/index.html:1]: in general great PR! the code is very readable and the logic is split into small and reusable parts 👏
few functional / bugs I've noticed:

- make sure "enter" key also adds a new todo (not just clicking the button)
- Clear input after a new todo is added
- Uncheck don’t Clear the strike through
- The edit is not clear. You take what's currently in the input.. think of a better UX

**Review by ofirdagan — CHANGES_REQUESTED** [review]

**Review by tombenezra — APPROVED** [review]

File: 1-Vanilla/index.html (hunk 6 above)
Comment (ofirdagan) [reply 1-Vanilla/index.html]: ?

File: 1-Vanilla/index.html (hunk 7 above)
Comment (ofirdagan) [reply 1-Vanilla/index.html]: ?

**Pull request merged**